README.md

```diff
@@ -70,13 +87,16 @@ If value matches pattern, binds variables and returns `value`. Otherwise, throws
 end
 ```
 
-Returns `result` for the first matching pattern. If there are no matching patterns, throws MatchFailure.
+Returns `result` for the first matching pattern. If there are no matching patterns, throws `MatchFailure`.
+
+Note that unlike the _assignment syntax_, this does not create any variable bindings outside the match macro.
```

Comment:
Is this true? I thought that it would re-assign to existing variables?

Reply:
I think only the assignment syntax does this right now.

See, for example, these tests:
https://github.com/RelationalAI-oss/Rematch.jl/blob/master/test/rematch.jl#L33-L39
vs these:
https://github.com/RelationalAI-oss/Rematch.jl/blob/master/test/rematch.jl#L310-L312

Reply:
Ah thanks for the clarification